cmd/agent/main_windows.go

```diff
@@ -47,14 +47,19 @@ func setupLogger(logLevel string) error {
 }
 
 func main() {
-	isIntSess, err := svc.IsAnInteractiveSession()
-	if err != nil {
-		fmt.Printf("failed to determine if we are running in an interactive session: %v", err)
-	}
-	if !isIntSess {
-		common.EnableLoggingToFile()
-		runService(false)
-		return
+	// if command line arguments are supplied, even in a non interactive session,
```

Comment:
This doesn't seem related to the PR topic, can you extend the description or remove unrelated changes?